src/main/js/index.tsx

```diff
@@ -101,6 +102,16 @@ const ShowPullRequestsRoute = ({ url, repository }: RepoRouteProps) => {
   );
 };
 
+const AllPullRequestsLink: FC = () => {
+  const [t] = useTranslation("plugins");
+  return <CardColumnSmall
+      link="/search/pullRequest/?q=status:OPEN"
+      contentLeft={t("scm-review-plugin.landingpage.myPullRequests.allPullRequestsLinkTitle")}
+      contentRight=""
+      avatar={<Icon name="noop" className="fa-fw fa-lg" />}
```

Comment:
I can't find an icon with the name noop. And the class fa-fw is not needed, because it is already set by the icon component.

Reply:
noop = no operation
its just used to get the left spacing equal to the PRs with avatar

Reply:
Ahh okay, I didn't know that abbreviation.